app/scripts/templates/settings/avatar_gravatar.mustache

```diff
@@ -6,7 +6,7 @@
   <section>
     <div class="avatar-wrapper with-default">
       {{#gravatar}}
-        <img src="{{ gravatar }}" id="gravatar"/>
+        <img src="{{ gravatar }}" id="gravatar" alt="Avatar" />
```

Comment:
nit: alt can be set to "Gravatar" as well. Most users are aware of what that means.